llvm/lib/Transforms/Scalar/CorrelatedValuePropagation.cpp

```diff
@@ -1032,6 +1033,24 @@ static bool processSExt(SExtInst *SDI, LazyValueInfo *LVI) {
   return true;
 }
 
+static bool processZExt(ZExtInst *SDI, LazyValueInfo *LVI) {
```

Comment:
```suggestion
static bool processZExt(ZExtInst *ZExt, LazyValueInfo *LVI) {
```
Or something? Not sure what SDI is supposed to stand for here.